common/src/main/java/com/wynntils/models/character/CharacterSelectionModel.java

```diff
@@ -25,20 +24,20 @@
 import org.lwjgl.glfw.GLFW;
 
 public final class CharacterSelectionModel extends Model {
-    private static final Pattern NEW_CLASS_ITEM_NAME_PATTERN = Pattern.compile("§l§a\\[\\+\\] Create a new character");
-    private static final Pattern CLASS_ITEM_NAME_PATTERN = Pattern.compile("§l§6\\[>\\] Select (.+)");
-    private static final Pattern CLASS_ITEM_CLASS_PATTERN =
-            Pattern.compile("§e- §r§7Class: (§r)?(§c§l☠§r)?(§6§l❂§r)?(§3§l⛏§r)?(§5§l⚔§r)?(\\s)?(§r)?§f(?<name>.+)");
-    private static final Pattern CLASS_ITEM_LEVEL_PATTERN = Pattern.compile("§e- §r§7Level: §r§f(\\d+)");
-    private static final Pattern CLASS_ITEM_XP_PATTERN = Pattern.compile("§e- §r§7XP: §r§f(\\d+)%");
-    private static final Pattern CLASS_ITEM_SOUL_POINTS_PATTERN = Pattern.compile("§e- §r§7Soul Points: §r§f(\\d+)");
+    private static final Pattern NEW_CLASS_ITEM_NAME_PATTERN = Pattern.compile("^§a\\[\\+\\] Create a new character$");
+    private static final Pattern CLASS_ITEM_NAME_PATTERN = Pattern.compile("^§6\\[>\\] Select (.+)$");
+    private static final Pattern CLASS_ITEM_CLASS_PATTERN = Pattern.compile(
+            "§e- §7Class: (§r)?(§c§l☠)?(§r)?(§6§l❂)?(§r)?(§3§l⛏)?(§r)?(§5§l⚔)?(§r)?(\\s)?(§r)?§f(?<name>.+)");
```

Comment:
Note that this pattern is not optimal, and not fully tested. I have no special mode characters, so I can't test the formatting of those markers. I think I played it safe by making the §r optional, so I'm guessing/hoping this will work.

The other patterns have been tested/confirmed.